ci/README.md

```diff
@@ -43,8 +43,10 @@ These issues effectively list PRs the merge bot has interacted with.
 To ensure security and a focused utility, the bot adheres to specific limitations:
 
 - The PR targets `master`, `staging`, or `staging-next`.
-- The PR only touches files located under `pkgs/by-name/*`.
-- The PR is authored by [@r-ryantm](https://nix-community.github.io/nixpkgs-update/r-ryantm/) or a [committer][@NixOS/nixpkgs-committers].
+- The PR only touches packages located under `pkgs/by-name/*`.
+- The PR is either:
+  - approved or authored by a [committer][@NixOS/nixpkgs-committers], or
```

Comment:
```suggestion
  - approved by a [committer][@NixOS/nixpkgs-committers],
  - authored by a [committer][@NixOS/nixpkgs-committers], or
```

Reply:
Also: should it explicitly mention "the latest push is approved by a committer"?

Reply:
> Also: should it explicitly mention "the latest push is approved by a committer"?

I did not include that in the README. I think it's too much detail for what that tries to achieve. Feedback will be given, because the label will be removed once the approval doesn't count anymore - so people will be able to figure this out on their own.